src/MovingHarmonicPotential.cc

```diff
@@ -58,15 +53,17 @@ namespace detail
 /*!
  * \param m Python module to export to
  */
-void export_ImplicitEvaporator(pybind11::module& m)
+void export_MovingHarmonicPotential(pybind11::module& m)
     {
     namespace py = pybind11;
-    py::class_<ImplicitEvaporator, std::shared_ptr<ImplicitEvaporator>>(m,
-                                                                        "ImplicitEvaporator",
-                                                                        py::base<ForceCompute>())
+    py::class_<MovingHarmonicPotential, std::shared_ptr<MovingHarmonicPotential>>(m,
+                                                                                  "MovingHarmonicPotential",
+                                                                                  py::base<ForceCompute>())
         .def(py::init<std::shared_ptr<SystemDefinition>, std::shared_ptr<Variant>>())
-        .def("setParams", &ImplicitEvaporator::setParams);
+        .def("setParams", &MovingHarmonicPotential::setParams);
```

Comment:
To work with the per-type Python interface, I think this is going to need to be modified to accept a data structure for this class that contains all the parameters. That data structure is also going to need a constructor that creates it from a Python dictionary (and I think one that can go the other way too).

Take a look at the methods here for what this class needs as an example:

https://github.com/glotzerlab/hoomd-blue/blob/58b7ba503cadd90cc1ed65f40208a8d2de5b7b63/hoomd/md/PotentialExternal.h#L185

and here for how the parameter class gets implemented:

https://github.com/glotzerlab/hoomd-blue/blob/58b7ba503cadd90cc1ed65f40208a8d2de5b7b63/hoomd/md/EvaluatorExternalMagneticField.h#L48-L76